bin/do_cf.py

```diff
@@ -101,6 +101,10 @@ def corr_func(p):
 
     cosmo = constants.cosmo(args.fid_Om)
 
+    lambda_abs  = constants.absorber_IGM[args.lambda_abs]
+    if (args.lambda_abs2) : lambda_abs2 = constants.absorber_IGM[args.lambda_abs2]
+    else: lambda_abs2 = constants.absorber_IGM[args.lambda_abs]
```

Comment:
please remove this line. if no input it should be None

Reply:
If you want to compute the cross-correlation for Lya in lya and lya in lyb. You need to have something for lambda-abs2. 
If I remove that, I will have to write "--lambda-abs2" everytime I want to compute a xcf.

Reply:
> If I remove that, I will have to write "--lambda-abs2" everytime I want to compute a xcf.

It is logical to have to write `--lambda-abs2` everytime you compute a cross-forest-forest correlation
It is not logical to write `--lambda-abs2` when you compute an auto-forest correlation.

Reply:
I agree but here you don't write it for the auto-correlation

Reply:
Just test doing the auto-correlation of SiIV or CIV, you will see the problem

Reply:
I've tested it 
command : "do_cf.py --in-dir deltas-civ --lambda-abs "CIV(1551)" --out auto_civ.fits "
I don't understand your problem. Did you use the last version of this branch ?

Reply:
perfect, if it works and if it is not flagged as cross as before, it is perfect.